example/README.md

```diff
@@ -0,0 +1,16 @@
+# example_app
+
+A new Flutter project.
```

Comment:
Delete this boilerplate text and Add something simple like "Showcase app to shows how the package works"